src/validate.ts

```diff
@@ -23,19 +23,19 @@ type customDatas = {[key: string]: any}
 interface QueryRequest{
     requestId: string;
     inputs: {
-     intent: string; payload: { devices: { id: string; customData: customDatas; }[]; };
+     intent: string; payload: { devices: { id: string; CustomData: CustomData; }[]; };
```

Comment:
The variable name should remain camelCase, while the type is TitleCase.

`customData: CustomData`

Reply:
Also you should make sure that this compiles. Right now it won't because your type is still called `customDatas`.

Reply:
Compiles now!